test/e2e/util/util.go

```diff
@@ -368,28 +368,31 @@ func EnsureAPIBudget(t *testing.T, ctx context.Context, client crclient.Client,
 
 		// Compare metrics against budgets
 		namespace := manifests.HostedControlPlaneNamespace(hostedCluster.Namespace, hostedCluster.Name).Name
+		clusterAgeMinutes := time.Since(hostedCluster.CreationTimestamp.Time).Round(time.Minute)
```

Comment:
need `.Minutes()` after the `Round()`

Reply:
need to type cast the result as well
```
clusterAgeMinutes := int32(time.Since(hostedCluster.CreationTimestamp.Time).Round(time.Minute).Minutes())
```

Reply:
Ahhh! Thanks!